src/c_kzg_4844.c

```diff
@@ -790,8 +790,7 @@ static bool pairings_verify(const g1_t *a1, const g2_t *a2, const g1_t *b1, cons
     return blst_fp12_is_one(&gt_point);
 }
 
-
-typedef BLSFieldElement Polynomial[FIELD_ELEMENTS_PER_BLOB];
+typedef struct { BLSFieldElement elements[FIELD_ELEMENTS_PER_BLOB]; } Polynomial;
```

Comment:
I would maybe rename `elements` to `evals` or `evaluations` here since this is a polynomial in evaluation form.

I feel like `elements` is a vague term and `polynomial->element[i]` not actually meaningful in this context.

Reply:
Gotcha. That works for me! Didn't really know what to name it.